src/quest_manager/forms.py

```diff
@@ -81,9 +81,9 @@ class Meta:
         }
 
         widgets = {
-            'instructions': SummernoteInplaceWidget(),
-            'submission_details': SummernoteInplaceWidget(),
-            'instructor_notes': SummernoteInplaceWidget(),
+            'instructions': ByteDeckSummernoteSafeWidget(),
+            'submission_details': ByteDeckSummernoteSafeWidget(),
```

Comment:
`instructions` and `submission_details` fields should both use the AdvancedWidget